api/client/swarm/join_token.go

```diff
@@ -12,11 +12,6 @@ import (
 	"golang.org/x/net/context"
 )
 
-const (
-	flagRotate = "rotate"
-	flagQuiet  = "quiet"
-)
```

Comment:
I think it's best to put constants in the file that uses them if they're only used in one place, but I guess it's fine to move them.